Android/AmazonConnectInAppCallingAndroidSample/app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,7 @@
     <uses-permission android:name="android.permission.READ_PHONE_STATE" />
     <uses-permission android:name="android.permission.CALL_PHONE" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE_MICROPHONE" />
```

Comment:
Is this a new permission needed for Android 15? If yes, we should also update Chime SDK Android readme.